lib/pbench/test/unit/server/test_datasets_metadata.py

```diff
@@ -29,22 +29,44 @@ def test_get_bad_keys(self, client, server_config, attach_dataset):
         )
         assert response.status_code == HTTPStatus.BAD_REQUEST
         assert response.json == {
-            "message": "Unrecognized list values ['xyzzy', 'plugh'] given for parameter metadata; expected saved,seen,user,deletion,access,owner"
+            "message": "Unrecognized list values ['plugh', 'xyzzy'] given for parameter metadata; expected dashboard.saved,dashboard.seen,user.,server.deletion,access,owner"
```

Comment:
Yeah, I thought about that for almost a billionth of a second before moving on to something else. Maybe `user.*` would be better. My first reaction was that it would complicate the code, but I suppose the only real change would be `f"{key.split('.')}.*"` for the match term. It'd be even easier if I went to a `Keyword` object, since it just means the `__str__` would add `".*"` if the namespace attribute was set. So ... hmm.